NitroxServer/ConsoleCommands/SaveCommand.cs

```diff
@@ -1,23 +1,73 @@
-﻿using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.ConsoleCommands.Abstract;
 using NitroxModel.DataStructures.GameLogic;
 using NitroxModel.DataStructures.Util;
+using NitroxModel.Logger;
+using System;
 
 namespace NitroxServer.ConsoleCommands
 {
+    public enum SaveMode{
+        ONLYAUTO,
+        ONLYEXIT,
+        NORMAL,
+        MANUAL
+    }
     internal class SaveCommand : Command
     {
-        public SaveCommand() : base("save", Perms.ADMIN)
+        public static SaveMode saveMode = SaveMode.NORMAL;
+
+        public SaveCommand() : base("save", Perms.ADMIN, "<optional:option (type save help or save -h for options)>", "Saves the server status. (Or optionally configures auto-save.)", new[] {"saveconfig", "configsave", "autosaveoptions"})
         {
         }
 
         public override void RunCommand(string[] args, Optional<Player> player)
         {
-            Server.Instance.Save();
+            if((args.Length == 0)||(args[0] == "now")||(args[0] == "-n")){
+                Server.Instance.Save();
+                Log.InfoIG("Server state saved!");
+            }
+            else{
+                if(args[0] == "onlyauto"){
```

Comment:
Some if branches to similar code. Consider joining the checks together to have less duplicate code:

```
if (args[0] == "test" || args[0] == "other-test") 
{
   // Same code for both cases.
}
```